core/src/com/unciv/logic/map/MapUnit.kt

```diff
@@ -447,7 +449,10 @@ class MapUnit {
 
     fun getFortificationTurns(): Int {
         if (!isFortified()) return 0
-        return action!!.split(" ")[1].toInt()
+        // temporary backwards compatibility
```

Comment:
Actually, the "p(r)op(p)er" backwards compatibility would be an all-units scan called from GameInfo.setTransients and homed in `object BackwardCompatibility` -> easier to find out-aged ones. Just saying. Sorry I didn't read your commits earlier. And there I'd probably `Regex("""^Fortify\s+(\d+)""")` and use the captured group or so.